server/routes.go

```diff
@@ -2,13 +2,21 @@ package server
 
 import (
 	_ "main/docs"
```

Comment:
why `_` here ?

Reply:
cleared sir

Reply:
Actually _ in front of import statement is used to keep them safe from auto save (vscode remove the import packages whose related code is not directly written in file but sometimes we need these packages indirectly to run some stuff like swagger doc or testing packages ).

Reply:
interesting, have not seen this yet